pkg/driver/controller.go

```diff
@@ -119,6 +119,11 @@ func (cs *controller) init() error {
 	go cs.k8sNodeInformer.Run(stopCh)
 	go cs.zfsNodeInformer.Run(stopCh)
 
+	if zfs.GoogleAnalyticsEnabled == "true" {
```

Comment:
Does this send an event everytime the controller starts?

Reply:
Yup, seems so.